pages/api/self-check/delete.ts

```
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { SelfCheckGroup } from "../models/selfCheckGroup";
+
+const delSelfCheckByID = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const { id } = req.query;
+    console.log(id);
+    let selfCheck;
+    try {
+        selfCheck = await SelfCheckGroup.findByIdAndDelete(id).exec();
+    } catch (err) {
+        console.log("error is ", err);
```

lets return an error here instead of just doing a console.log